lib/beaker/version.rb

```diff
@@ -1,5 +1,5 @@
 module Beaker
   module Version
-    STRING = '4.41.2'
+    STRING = '4.41.3'
```

Comment:
this this release contains an enhancement it has to be a minor release. Please make it 4.42.0

Reply:
Updated the release version to 4.42.0